### PR TITLE
chore: manual release process + npm audit fix

### PR DESCRIPTION
**Pull request by iuricmp**
Signed-off-by: Iuri Pereira <689440+iuricmp@users.noreply.github.com>

### CODE REVIEW

File: mobile/app.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "name": "dsocial",
     "slug": "dsocial",
     "platforms": ["ios", "android"],
-    "version": "1.0.0",
+    "version": "1.9.3",
```
Comment (D4ryl00) [inline mobile/app.json:6]: We shouldn't change the version number here because it won't be updated automatically.

File: mobile/app.json [hunk 2]
```diff
@@ -28,15 +28,15 @@
           }
         }
       },
-      "buildNumber": "9"
+      "buildNumber": "12"
```
Comment (D4ryl00) [inline mobile/app.json:31]: Regarding this field, I don't know yet if we have to keep it updated or not (I think it should be like "version", so no updated).

File: mobile/app.json [hunk 3]
```diff
@@ -28,15 +28,15 @@
...
     },
     "android": {
       "adaptiveIcon": {
         "foregroundImage": "./assets/images/adaptive-icon.png",
         "backgroundColor": "#ffffff"
       },
       "package": "tech.berty.dsocial.android",
-      "versionCode": "9"
+      "versionCode": "12"
```
Comment (D4ryl00) [inline mobile/app.json:39]: Same things here as above

File: mobile/README.md [hunk 4]
```diff
@@ -120,10 +120,24 @@ make asdf.install_tools
...
+1. Download the `GoogleService-Info.plist` file from the Firebase Console.
+2. Run `ios.release_production` to build the app.
+3. After the build is complete, submit it to the App Store running `eas submit --platform ios`
+
+### Google Play Store
+
+1. Download the `google-services.json` file from the Firebase Console.
+2. Run `android.release_production` to build the app.
+3. After the build is complete, submit it to the Play Store running `eas submit --platform android`
+You'll need to have a s[ervice account json file](https://developers.google.com/android/management/service-account) to authenticate with Google Play Store.
```
Comment (jefft0) [inline mobile/README.md:142]: "s[ervice" should be "[service"

File: mobile/README.md [hunk 5]
```diff
@@ -120,10 +120,24 @@ make asdf.install_tools
...
 # to start Metro Bundler:
 make start
 ```
 
+## Manual release to Google Play Store and Apple App Store
+
+### Apple App Store
+
+1. Download the `GoogleService-Info.plist` file from the Firebase Console.
+2. Run `ios.release_production` to build the app.
```
Comment (jefft0) [inline mobile/README.md:134]: `ios.release_production` should be `make ios.release_production`

File: mobile/README.md [hunk 6]
```diff
@@ -120,10 +120,24 @@ make asdf.install_tools
...
+3. After the build is complete, submit it to the App Store running `eas submit --platform ios`
+
+### Google Play Store
+
+1. Download the `google-services.json` file from the Firebase Console.
+2. Run `android.release_production` to build the app.
```
Comment (jefft0) [inline mobile/README.md:140]: `android.release_production` should be `make android.release_production`

File: mobile/app.json (hunk 1 above)
Comment (jefft0) [reply mobile/app.json:6]: https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md

File: mobile/app.json (hunk 2 above)
Comment (jefft0) [reply mobile/app.json:31]: https://github.com/semantic-release/semantic-release/blob/master/docs/support/FAQ.md

**Pull request merged**